detect_secrets/core/secrets_collection.py

```diff
@@ -363,11 +365,17 @@ def _extract_secrets_from_file(self, f, filename):
                 )
                 f.seek(0)
 
-        except UnicodeDecodeError as error:
-            log.warning(
-                '%s failed to load and could not be scanned.\nError: %s',
-                filename, str(error),
-            )
+        except UnicodeDecodeError:
+            if (is_binary(filename)):
+                # If the file type is binary, suppress the warning
+                pass
+            else:
+                log.warning(
```

Comment:
Since this **does not** come out for **binary** files; it should not be to extraneous .. I think it is a informative message.  I like it.